collectors/modality-probe-collector-common/src/lib.rs

```diff
@@ -134,20 +141,134 @@ impl From<EventLogEntry> for LogEntryData {
 
 /// A row in a collected trace.
 #[derive(Debug, serde::Serialize, serde::Deserialize, Eq, PartialEq)]
-struct LogFileRow {
-    session_id: u32,
-    sequence_number: u64,
-    sequence_index: u32,
-    receive_time: DateTime<Utc>,
-    probe_id: u32,
-    fc_probe_id: Option<u32>,
-    fc_probe_epoch: Option<u16>,
-    fc_probe_clock: Option<u16>,
-    event_id: Option<u32>,
-    event_payload: Option<u32>,
-    tc_probe_id: Option<u32>,
-    tc_probe_epoch: Option<u16>,
-    tc_probe_clock: Option<u16>,
+pub struct LogFileRow {
+    pub session_id: u32,
+    pub sequence_number: u64,
+    pub sequence_index: u32,
+    pub receive_time: DateTime<Utc>,
+    pub probe_id: u32,
+    pub fc_probe_id: Option<u32>,
+    pub fc_probe_epoch: Option<u16>,
+    pub fc_probe_clock: Option<u16>,
```

Comment:
Is there a reason we have ungrouped Option fields as opposed to semantically-linked groups? E.G.`pub fc: Option<FrontierClock>,` rather than 3 flat fields (now that we're not in csv-land?).

Actually, if we're okay with nested structure now, how about using an actual enum for switching between the 3 main content variants?

`enum RowContents { FrontierClock(etc), Event(etc), TraceClock(etc) }`

Reply:
Nope! This type is actually not used anymore. It's going to come out altogether as soon as I have time to fix my graph tests that are still using it. I've a unchecked item on the PR to remind myself to do this. Everything is `ReportLogEntry` now, which does exactly that!

Reply:
Excellent, good to know.

Reply:
Recommend marking this struct as `#[cfg(test)]` in the meantime.